requirements.txt

```diff
@@ -17,3 +17,4 @@ mock
 pydotplus
 pydot>=1.2.3
 packaging
+geomstats>=1.10
```

Comment:
It sounds like we're not prepared to make this a full dependency, so what we can do is add this to an `EXTRA_REQUIRES` in [`nipype.info`](https://github.com/nipy/nipype/blob/master/nipype/info.py):

https://github.com/nipy/nipype/blob/3dc550035eaf91afc2d932a7e7bafe4871c94602/nipype/info.py#L159-L170

Of the existing extras, it probably fits best in `nipy`, and we can put a constraint on it so we don't try to install it in Python versions that don't have it:

```Python
        'nipy': ['nitime', 'nilearn', 'dipy', 'nipy', 'matplotlib',
                 'geomstats>=1.1.0; python_version >= "3.5"'],
```